frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -189,6 +203,13 @@ export function AccountsSummary({
                   <AddBuyReceiveOnEmptyBalances accounts={accounts} balances={balances} />
                 ) : undefined
               } />
+            {accountsByKeystore.length > 1 && (
+              <CoinBalance
+                accounts={accounts}
+                summaryData={summaryData}
+                coinsBalances={coinsTotalBalance}
+              />
+            )}
```

Comment:
The problem was in this part here where I returned `accountSummary.noAccount`, I updated to return `null` instead.
`
      return (
        <tr>
          <td colSpan={3} className={style.noAccount}>
            {t('accountSummary.noAccount')}
          </td>
        </tr>
      );
    })) : (
    <tr>
      <td colSpan={3} className={style.noAccount}>
        {t('accountSummary.noAccount')}
      </td>
    </tr>
`

Reply:
The problem was in the `<tbody>` in `CoinBalance` function where I returned `accountSummary.noAccount` but I think it will be better to return `null` there.